packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -113,63 +126,77 @@ To complete the one-time verification of the Event Hook:
 
 or
 
-* After making sure that your  ngrok session and local application are ready for the external verification call, go to the Event Hooks table, click the **Actions** drop-down menu of your **UNVERIFIED** Event Hook, and select **Verify**.
+* After making sure that your  ngrok session and local application are ready for the external verification call, go to the Event Hooks table, click the **Actions** dropdown menu of your **UNVERIFIED** Event Hook, and select **Verify**.
 
 The Event Hook is now set up with a status of **VERIFIED** and is ready to send Event Hook calls to your external service.
 
 >**Note:** A successful Event Hook verification also indicates your local application is working with the ngrok session! Review the ngrok terminal or inspector interface for details on the first `GET` call to your local application.
 
-## Preview, test, and review the Event Hook
+## Test, preview, and review the Event Hook
 
-With your local application now exposed externally through an ngrok session, you can preview and test Okta Event Hook calls, as well as review details of the calls using the ngrok inspection interface. The Okta org is also set up to call your local application when an event is triggered. In this example, the event triggers when a user is added to your Okta org.
+With your local application now exposed externally through an ngrok session, you can test and preview Okta Event Hook calls, as well as review details of the calls using the ngrok inspection interface. The Okta org is also set up to call your local application when an event is triggered. In this example, the event triggers when a user is added to your Okta org.
 
-### Preview
+### Test the Event Hook
 
-To run a preview call of your Event Hook:
+To run a test of your Event Hook:
 
+1. Ensure that both your ngrok session and local sample application are running.
 1. In your Okta org, sign in as an administrator and create a test user in the Admin Console.
-    * Go to **Directory** > **People**, and click **Add Person**. As an example, add the user John Doe with the following fields:
+    * Go to **Directory** > **People**, and click **Add Person**. As an example, add the user Jane Doe with the following fields:
```

Comment:
```suggestion
    * Go to **Directory** > **People**, and click **Add Person**. As an example, add the user John Doe with the following fields:
```

Reply:
The example has "John" instead of "Jane"?

Reply:
Thanks, good catch.